src/Skoruba.IdentityServer4.Admin/Helpers/Messaging/SendgridEmailSender.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Extensions.Logging;
```

Comment:
This would be good to have in a shared project between both the Admin and the STS.Identity project.  This was pulled in from the STS.Identity project.